test/framework/module_generator.py

```diff
@@ -139,6 +141,57 @@ def test_descr(self):
         desc = self.modgen.get_description()
         self.assertEqual(desc, expected)
 
+    def test_set_default(self):
+        """
+        Test load part in generated module file.
+        """
+        if self.MODULE_GENERATOR_CLASS == ModuleGeneratorLua and not isinstance(self.modtool, Lmod):
+            pass
```

Comment:
@victorusu: you'll need to make this a `return` rather than a `pass` to actually skip the test